scraper/scraper.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 The Prometheus Authors
```

Comment:
Minor nit, when making new files, we set the current year. `2022`.

Reply:
Done